drivers/timer/stm32_lptim_timer.c

```diff
@@ -27,6 +27,9 @@
 
 #define LPTIM (LPTIM_TypeDef *) DT_INST_REG_ADDR(0)
 
+static const struct stm32_pclken lptim_clk[] = STM32_DT_INST_CLOCKS(0);
+static const struct device *clk_ctrl = DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE);
```

Comment:
nit: check for readiness if https://github.com/zephyrproject-rtos/zephyr/pull/48787 goes first

Reply:
Thanks, done